nameko_grpc/dependency_provider.py

```diff
@@ -0,0 +1,143 @@
+from nameko.extensions import DependencyProvider
```

Comment:
Probably would want to add `flake8` checks on the build. Just this file:
```
nameko_grpc/dependency_provider.py:3:1: F401 'select' imported but unused
nameko_grpc/dependency_provider.py:5:1: F401 'eventlet.event.Event' imported but unused
nameko_grpc/dependency_provider.py:6:1: F401 'eventlet' imported but unused
nameko_grpc/dependency_provider.py:9:1: F401 'h2.errors.PROTOCOL_ERROR' imported but unused
nameko_grpc/dependency_provider.py:61:80: E501 line too long (88 > 79 characters)
nameko_grpc/dependency_provider.py:97:80: E501 line too long (80 > 79 characters)
nameko_grpc/dependency_provider.py:129:80: E501 line too long (86 > 79 characters)
```

Reply:
I've added pre-commit hooks with flake8 now 👍